bran/helpers/raven.py

```diff
@@ -0,0 +1,165 @@
+from bran.helpers.utils import get_security_groups, get_local_awsconfig, list_to_choices, get_comet_api_key
+import git
+import os
+
+
+def get_raven_branches():
+    """
+        Returns a list of branch names in ravenML-train-plugins
+        
+        Return:
+            remote_refs (list): list of ravenML-train-plugins branch names
+    """
+    url = "https://github.com/autognc/ravenML-train-plugins"
```

Comment:
Is this intended to be used by outside people as well? If so it maybe handy to have an option for the repo itself (which could just default to ours) since they prob will be on a fork.

Reply:
Totally agree. Also dynamic plugin options depending on which branch the user selects (comet-opt, tune branches have unique plugins) would be nice